packages/adp-tooling/test/unit/writer/index.test.ts

```diff
@@ -86,5 +87,10 @@ describe('ADP writer', () => {
                 )
             ).toMatchSnapshot();
         });
+        test('migrate minimal config', async () => {
```

Comment:
Could you please put this into a new describe block. We usually have a describe for the whole file and then nested describes for each function we test, and in there the specific test cases.

For the test itself, could you please add a fixture containing a simple "old" project and then run the migrate command?

Reply:
Please remove the `pom.xml` from your example. That is only required internally, so it doesn't need to be here for a test.

Reply:
Shouldn't your migrate function also delete the `xs-app.json` and the `.che` folder?

Reply:
Thanks again, I have updated the code.

Reply:
> Shouldn't your migrate function also delete the `xs-app.json` and the `.che` folder?

I think we should keep that separate along with other migration logic. Here we would only generate the needed adp template files.

Reply:
Code has been updated to remove `.che` and `neo-app.json`. 👍